railties/lib/rails/generators/rails/app/templates/config/puma.rb

```diff
@@ -0,0 +1,9 @@
+threads_count = ENV.fetch("MAX_THREADS") { 5 }.to_i
```

Comment:
@schneems If you're not likely to change these values per-environment, why do you need to be able to overwrite them ad-hoc via ENV?